File: src/main/java/kr/nyamnyam/model/domain/ReplyModel.java
package kr.nyamnyam.model.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;

import java.time.LocalDateTime;

@Component
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReplyModel {
    private Long id;
    private String content;

    private LocalDateTime entryDate;
    private LocalDateTime modifyDate;

    private Long postId;

    private String userId;
    private String nickname;

}

File: src/app/model/reply.model.ts
export interface ReplyModel{
    id:number;
    content:string;
    postId: number;
    userId: number;
    nickname: string;
    entryDate: string;
    modifyDate?: string;
}

export const initialReply: ReplyModel = {
    id: 0,
    content: '',
    postId: 0,
    userId: 0,
    nickname: '',
    entryDate: '',
    modifyDate: '',
}